Modules/Services/Sources/Models/Extensions/DetailsLayoutExtension.swift

```diff
@@ -9,12 +9,13 @@ public extension DetailsLayout {
     static let mediaLayouts: [DetailsLayout] = [ .image, .audio, .video ]
     static let fileAndMediaLayouts = DetailsLayout.fileLayouts + DetailsLayout.mediaLayouts
     
-    static let visibleLayouts: [DetailsLayout] = listLayouts + editorLayouts + [.bookmark] + [.participant] + dateLayout + chatLayout
+    static let visibleLayouts: [DetailsLayout] = listLayouts + editorLayouts + [.bookmark, .participant, .objectType] + dateLayout + chatLayout
```

Comment:
maybe we should add toggle for objectType?

Reply:
The issue is that we cannot toggle the AllContentType enum. Therefore, in an emergency, it will require manual disabling of this feature, and toggling does not really make sense in this case.

Reply:
I mean toggle adding to `visibleLayouts` while the objectType page is not ready